src/test/java/api/items/ItemApiExamples.java

```diff
@@ -114,6 +118,13 @@ public void canCreateAnItemWithoutIDAndHRID()
 
     assertThat(createdItem.containsKey("id"), is(true));
 
+    assertThat(createdItem.containsKey("administrativeNotes"), is(true));
+
+    List<String> createdNotes = createdItem.getJsonArray("administrativeNotes")
+      .stream().map(arrayItem -> arrayItem.toString()).collect(Collectors.toList());
```

Comment:
The utility functions in the [JsonArrayHelper](https://github.com/folio-org/mod-inventory/blob/2c3e46cd82d31587d711e6d31fd3407d2aae3b7e/src/main/java/org/folio/inventory/support/JsonArrayHelper.java#L13) might help with this.